backend/src/main/resources/application.properties

```diff
@@ -1,8 +1,22 @@
-spring.datasource.url=jdbc:mysql://localhost:3306/SkyNest?createDatabaseIfNotExist=true
+
+#   MySQL
+spring.datasource.url=jdbc:mysql://localhost:3305/SkyNest?createDatabaseIfNotExist=true
 spring.datasource.username=user
 spring.datasource.password=123456
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL5InnoDBDialect
 spring.flyway.baseline-on-migrate=true
 #spring.jpa.hibernate.ddl-auto=none
```

Comment:
Commented code should never be part of a commit, either uncomment it or remove it

Reply:
Will remove.